model/src/test/test_model_basic_renewal.py

```diff
@@ -33,16 +33,21 @@ def test_model_basicrenewal_no_timepoints_or_observations():
         name="gen_int", value=jnp.array([0.25, 0.25, 0.25, 0.25])
     )
 
-    I0 = DistributionalRV(name="I0", distribution=dist.LogNormal(0, 1))
+    I0_init_rv = InfectionInitializationProcess(
+        "I0_initialization",
+        DistributionalRV(name="I0", distribution=dist.LogNormal(0, 1)),
+        InitializeInfectionsZeroPad(n_timepoints=gen_int.basevar.value.size),
```

Comment:
Why this change throughout?

Reply:
I'm not sure why/how it was working before (if it was) but it shouldn't have, since `gen_int` is a `DeterministicVariable` and therefore doesn't have a defined `.size`.

Reply:
Can be reverted. Didn't realize `DeterministicPMF` variables had a `size()` method. Whether they should is a different question, but it's beyond the scope of this PR.

Reply:
Fixed in 1b34490